virtual_rainforest/core/utils.py

```diff
@@ -26,21 +27,19 @@ def extract_model_time_details(
         A tuple containing the start date and the update interval for the model
 
     Raises:
-        pint.errors.DimensionalityError: If the desired conversion between units isn't
-            possible, e.g. kg to s.
-        pint.errors.UndefinedUnitError: If the unit is not known to pint.
+        InitialisationError: If the model timing cannot be properly extracted
     """
 
     try:
         raw_interval = pint.Quantity(config[model_name]["model_time_step"]).to(
             "seconds"
         )
         # Round raw time interval to nearest minute
-        update_interval = timedelta64(int(round(raw_interval.magnitude)), "s")
+        update_interval = timedelta64(round(raw_interval.magnitude), "s")
 
         start_date = datetime64(config["core"]["timing"]["start_date"])
     except (pint.errors.DimensionalityError, pint.errors.UndefinedUnitError) as excep:
         LOGGER.error("Model timing error: %s" % str(excep))
-        raise excep
+        raise InitialisationError()
```

Comment:
I'd keep this info as an inner exception:

```suggestion
        raise InitialisationError() from excep
```